bin/build.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# Builds and uploads:
+#   - CLI to S3
+#   - Spark docker image to dockerhub
+#   - stub universe zip to S3
+#
+# Manifest config:
+#   CLI_VERSION - version label to use for CLI package
+#   SPARK_DIST_URI - where fetch spark distribution from
+#
+# ENV vars:
+#   COMMONS_TOOLS_DIR - path to dcos-commons/tools/, or empty to fetch latest release tgz
+#   DOCKER_IMAGE - "<image>:<version>", falls back to mesosphere/spark-dev:COMMIT)
+#   ghprbActualCommit / GIT_COMMIT - COMMIT value to use for DOCKER_IMAGE, if DOCKER_IMAGE isn't specified
+
+set +e
```

Comment:
If a build fails, Github will automatically mark the build result as failed, so I'm not sure we need this.  I'd prefer `set -e`